### PR TITLE
chore: add BCR to C03 project jsonld example

### PR DESCRIPTION
**Pull request by blushi**
## Description

ref: regen-network/regen-registry#1606

We should add Be Zero Carbon rating value and link to project anchored metadata, see examples in https://www.notion.so/regennetwork/8c3d639f3da24ffe8a7cd81db5adcd30?v=c2f8ac9017814e4580f6b6b1e322be17

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] reviewed "Files changed" and left comments if necessary

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] reviewed documentation is accurate
- [ ] manually tested (if applicable)

### CODE REVIEW

File: jsonld/projects/C03-project.jsonld [hunk 1]
```diff
@@ -43,5 +43,9 @@
     ""
   ],
   "regen:projectStartDate": "",
-  "regen:projectEndDate": ""
+  "regen:projectEndDate": "",
+  "regen:beZeroCarbonRating": {
```
Comment (blushi) [inline jsonld/projects/C03-project.jsonld:47]: only added this example for now, waiting for others feedback before moving forward with the SHACL schema updates

File: jsonld/projects/C03-project.jsonld [hunk 2]
```diff
@@ -43,5 +43,9 @@
...
+    "schema:value": "",
```
Comment (blushi) [inline jsonld/projects/C03-project.jsonld:48]: e.g. "AA-"
I was debating whether to use https://schema.org/ratingValue instead or not... but I think be zero carbon rating might be too specific to be considered just as a `schema:ratingValue` which seems to be used for rating products or places for instance
the alternative being:
```json
"regen:beZeronCarbon": {
  "schema:ratingValue": "AA-",
  "schema:url": "https://bezerocarbon.com/..."
}
```

Comment (blushi) [reply jsonld/projects/C03-project.jsonld:48]: it would make sense if we expected other kinds of third-party ratings than BeZero Carbon ratings, do you think we will @erikalogie?
also we need to think of a structure that will allow us to display it easily this way on the project page in the future (for now, it will only appear in the "additional info" section): https://www.figma.com/file/seUiQM2SPt78KJiDL0Ea8m/Basic-Ecocredits-Ledger-View-%2F-Marketplace?node-id=3495-166691&t=AgavpAHXkCl5CZxo-0 (although this design is still not finalized...)
within `schema:name` value here, there's no reason to use camel case, we could just use the actual name in plain english: "BeZero Carbon"

Comment (wgwz) [reply jsonld/projects/C03-project.jsonld:48]: > it would make sense if we expected other kinds of third-party ratings than BeZero Carbon ratings

yup, this was my thinking as well.

> within schema:name value here, there's no reason to use camel case, we could just use the actual name in plain english: "BeZero Carbon" but I'm thinking using @type instead of schema:name might be more appropriate in this case, wdyt?

this sounds good to me as well. i don't have a strong opinion about using the `@type` parameter here at the moment. for example, that could be useful if the "beZeroRating" was going to be subject to some different conditions that other rating types. but if agree that all `regen:rating`'s will have a name, ratingValue and a url, and that's all we will need, then maybe we don't need the `@type`. but did you have another use in mind for the type?

Comment (erikalogie) [reply jsonld/projects/C03-project.jsonld:48]: > it would make sense if we expected other kinds of third-party ratings than BeZero Carbon ratings, do you think we will @erikalogie? also we need to think of a structure that will allow us to display it easily this way on the project page in the future (for now, it will only appear in the "additional info" section): https://www.figma.com/file/seUiQM2SPt78KJiDL0Ea8m/Basic-Ecocredits-Ledger-View-%2F-Marketplace?node-id=3495-166691&t=AgavpAHXkCl5CZxo-0 (although this design is still not finalized...) within `schema:name` value here, there's no reason to use camel case, we could just use the actual name in plain english: "BeZero Carbon" but I'm thinking using `@type` instead of `schema:name` might be more appropriate in this case, wdyt?

It is possible that we will have other third party ratings yes

Comment (blushi) [reply jsonld/projects/C03-project.jsonld:48]: the idea behind using @type over just `schema:name` is that if we want to attach some other data to it in the UI, like a logo or a description that could be defined from Sanity for instance and the value of `@type` would be referenced there
ie, we could use the same pattern that we use currently for SDGs or Impact where we would have the IRI referenced in Sanity
that being said, I think we could also just use both `@type` and `schema:name`:

```json
"regen:rating": {
  "@type": "regen:BeZeroCarbon",
  "schema:name": "BeZero Carbon",
  "schema:ratingValue": "AA-",
  "schema:url": "https://bezerocarbon.com/..."
}
```

Comment (blushi) [reply jsonld/projects/C03-project.jsonld]: thoughts @wgwz?
it'd be cool to come to a conclusion asap so I can generate the metadata IRI and update the toucan projects metadata accordingly by eod

Comment (wgwz) [reply jsonld/projects/C03-project.jsonld]: LGTM!

File: ops/C03/project-metadata/C03-001-project.jsonld [hunk 3]
```diff
@@ -0,0 +1,71 @@
+{
+  "@type": "regen:C03-Project",
```
Comment (wgwz) [inline ops/C03/project-metadata/C03-001-project.jsonld:2]: just noting that for the C02, we included the `@id` parameter in the project and project-page metadata. not assuming we should be doing that here. just wanted to point it out. i don't think it's a critical issue by any means. and for these C03 projects it might be quite hard to populate that over the bridge 🤔

File: ops/C03/project-metadata/C03-003-project.jsonld [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  "regen:approvedMethodologies": {
+    "@type": "schema:ItemList",
+    "schema:itemListElement": [
+      {
+        "schema:name": "VM0015",
+        "schema:identifier": "VM0015"
+      }
+    ]
+  },
+  "schema:location": {
```
Comment (wgwz) [inline ops/C03/project-metadata/C03-003-project.jsonld:26]: just noting that we'll need to add this into the C03-project schema..

File: ops/C03/project-metadata/C03-001-project.jsonld (hunk 3 above)
Comment (blushi) [reply ops/C03/project-metadata/C03-001-project.jsonld:2]: thanks for the reminder, yeah I think we had discussed that it's not really possible to do that over the bridge, although I could have done it here as part of those manual updates, let's try to add it next time an update is needed I guess but I think it's fine if we don't have it yet

File: ops/C03/project-metadata/C03-003-project.jsonld (hunk 4 above)
Comment (blushi) [reply ops/C03/project-metadata/C03-003-project.jsonld:26]: I haven't updated the SHACL schemas yet, still TODO

File: shacl/projects/C03-project.ttl [hunk 5]
```diff
@@ -23,4 +25,30 @@ regen:C03-ProjectShape a sh:NodeShape ;
   sh:property regen:ProjectTypeOptionalPropertyShape ;
   sh:property regen:ProjectOptionalStartDatePropertyShape ;
   sh:property regen:ProjectOptionalEndDatePropertyShape ;
+
+  sh:property [
```
Comment (blushi) [inline shacl/projects/C03-project.ttl:29]: we could extract this later when it'll be on projects from other credit classes

File: shacl/projects/C03-project.ttl [hunk 6]
```diff
@@ -23,4 +25,30 @@ regen:C03-ProjectShape a sh:NodeShape ;
...
+    sh:path regen:rating ;
+    sh:node regen:BeZeroCarbon ;
```
Comment (blushi) [inline shacl/projects/C03-project.ttl:31]: as soon as we support more types of ratings, we could add some `sh:or` constraint here I guess instead

**Review by wgwz — APPROVED** [review]
tACK
